service/src/test/java/uk/gov/hmcts/reform/fpl/model/DirectionsTest.java

```diff
@@ -31,21 +33,28 @@
 
     @Test
     void shouldGetListOfDirectionsWithPopulatedCCDFieldsFromIndividualDirectionFields() {
-        CaseData caseData = CaseData.builder()
-            .directionsForCaseManagementOrder(Directions.builder()
-                .allPartiesCustomCMO(wrapElements(Direction.builder().build()))
-                .localAuthorityDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .respondentDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .cafcassDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .otherPartiesDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .courtDirectionsCustomCMO(wrapElements(Direction.builder().build()))
-                .build())
+        Directions directions = Directions.builder()
+            .allPartiesCustomCMO(wrapElements(Direction.builder().build()))
+            .localAuthorityDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .respondentDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .cafcassDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .otherPartiesDirectionsCustomCMO(wrapElements(Direction.builder().build()))
+            .courtDirectionsCustomCMO(wrapElements(Direction.builder().build()))
             .build();
 
         List<Element<Direction>> expected = wrapElements(getDirection(ALL_PARTIES), getDirection(LOCAL_AUTHORITY),
             getDirection(PARENTS_AND_RESPONDENTS), getDirection(CAFCASS), getDirection(OTHERS), getDirection(COURT));
 
-        assertThat(caseData.getDirectionsForCaseManagementOrder().getDirectionsList()).isEqualTo(expected);
+        assertThat(directions.getDirectionsList()).isEqualTo(expected);
+        assertTrue(directions.containsDirections());
+    }
+
+    @Test
+    void shouldReturnEmptyListWhenNoDirections() {
+        Directions directions = Directions.builder().build();
+
+        assertThat(directions.getDirectionsList()).isEmpty();
+        assertFalse(directions.containsDirections());
```

Comment:
minor: do we really need this assert? This is because `Directions.builder().build()` will have all fields as `null`.

Reply:
same as above, separate methods and should both have assertions